src/test/testing_assert.hpp

```diff
@@ -139,3 +139,34 @@ bool contained_in_query_plan(const std::shared_ptr<const AbstractOperator>& node
       GTEST_FAIL();                                              \
     }                                                            \
   }
+
+#define EXPECT_COLUMN_STATISTICS_IMPL(column_statistics_expr, null_value_ratio_, distinct_count_, min_, max_,       \
```

Comment:
We had the discussion about test macros a while ago (and didn't find a good solution). I think that this is too specific to be in `testing_assert.hpp` and that the interface is way too complicated to be used by anyone else but you.

Reply:
Ah, the reason why I put these in testing_assert is beacuase I use the same macros in generate_table_statistics_test.cpp as well. And I'll use them once I get to samples statistics generation as well.

So, I really need them, everything else just leads to (possibly incomplete, because it's easy to miss an attribute if you're checking them one by one) bloat-tests. A specific header file for these in test/statistics/, would that be okay with you?

Reply:
If we don't use it outside of `test/statistics`, that would be a good option. Still, we should revisit that at some point to clean up the interface.